config/locales/two_factor_authentication/en.yml

```diff
@@ -128,6 +128,9 @@ en:
       phone_info_html: Get your security code via text message (SMS) or phone call
       piv_cac: Government employees
       piv_cac_info_html: Use your PIV/CAC card to secure your account
+      second_phone: Second phone
+      second_phone_info_html: Get a security code by text message (SMS) or phone call
```

Comment:
I feel like `"Get a security code by text message (SMS) or phone call to a secondary phone number (one that is not %{phone})"` might flow better? 

Also, I talked to Michelle and she said the phone number should be masked, like `xxx-xxx-1234` (just forgot to add it to the story).

Reply:
Everything else looks good to me though!